src/library/features/baseplaylist/baseplaylistfeature.cpp

```diff
@@ -327,8 +340,22 @@ void BasePlaylistFeature::slotDeletePlaylist() {
         DEBUG_ASSERT_AND_HANDLE(playlistId >= 0) {
             return;
         }
-
+        
         m_playlistDao.deletePlaylist(playlistId);
+
+        // This avoids a bug where the m_lastChildClicked index is still a valid
```

Comment:
Yes, when I wrote this code I still didn't know about the `QPersistentModelIndexClass`